src/libpostal.c

```diff
@@ -1111,3 +1111,7 @@ void libpostal_teardown_parser(void) {
     geodb_module_teardown();
     address_parser_module_teardown();
 }
+
+const char* libpostal_version(void) {
+    return PACKAGE_VERSION;
+}
```

Comment:
1. Not sure this needs to be a function. A #define is more standard and it could e.g. be defined as a constant in the various bindings.
2. PACKAGE_VERSION is a bit autoconf-magic-y, not clear where it's defined for people not familiar with Autotools. Might be better to build a version header during configure